setup.py

```diff
@@ -376,17 +376,19 @@ def get_data_files():
     },
     'install_requires': [
         'async_generator',
-        'jupyter_server>=0.1.0,<0.2.0',
-        'nbconvert>=5.5.0,<6',
+        'jupyter_server>=0.3.0',
+        'jupyter_client>=6.1.3',
+        'nbclient>=0.2.0',
+        'nbconvert==6.0.0a1',
         'jupyterlab_pygments>=0.1.0,<0.2',
         'pygments>=2.4.1,<3'  # Explicitly requiring pygments which is a second-order dependency.
                               # An older versions is generally installed already and is otherwise not updated by pip.
     ],
     'extras_require': {
         'test': [
             'mock',
-            'pytest<4',
-            'pytest-tornado',
+            'pytest',
+            'pytest-tornasync',
```

Comment:
cool, but i wonder if we should follow jupyter-server by doing with pytest-asyncio.

Reply:
jupyter-server uses pytest-tornado too: https://github.com/jupyter/jupyter_server/blob/4f2d91d1754a30c9e46b48638c5a67907729fcb6/setup.py#L101